### PR TITLE
[strings] Add missing translations (historic)

### PR DESCRIPTION
**Pull request by j13m126**
I used wikidata, deepl and google translate to add missing translations to all historic POIs
Further:

- added missing pois to `categories.txt`
- sorted and added languages in `categories.txt`
- removed duplicate pt-BR translations

Signed-off-by: Lukas Kronberger <j13m126@posteo.de>

### CODE REVIEW

**Review by rtsisyk — APPROVED** [review]

File: data/categories.txt [hunk 1]
```diff
@@ -3046,20 +3121,77 @@ vi:Cổng thành
 zh-Hans:城门
 zh-Hant:城門
 
+historic-citywalls|@tourism
+en:City Walls
```
Comment (biodranik) [inline data/categories.txt:3125]: Should it be City Wall (single vs plural) here and in all derived translations?

File: data/categories.txt [hunk 2]
```diff
@@ -3046,20 +3121,77 @@ vi:Cổng thành
...
+ar:جدار المدينة
+be:Гарадскія сцены
```
Comment (biodranik) [inline data/categories.txt:3127]:
```suggestion
be:Гарадскі вал
```

File: data/categories.txt [hunk 3]
```diff
@@ -3288,39 +3495,40 @@ zh-Hant:戰爭紀念建築
 
 historic-monument|@tourism
 en:4Monument|U+1F5FD|U+1F5FF
-ru:4Монумент
-bg:4монумент|паметник
 ar:صرح|معالم|سياحة|مناظر
+be:Помнік
```
Comment (biodranik) [inline data/categories.txt:3499]:
```suggestion
be:4Помнік
```

File: data/categories.txt [hunk 4]
```diff
@@ -3361,69 +3569,188 @@ zh-Hant:刑柱
...
-sk:Zrúcanina|ruiny
-sw:Maeneo ya kale|jengo la kale|chakavu
-fa:خرابه های تاریخی
 
 historic-ship|@tourism
 en:Ship|boat
-ru:Корабль|судно|лодка
+ar:سفينة تاريخية
+be:Судна
 bg:Кораб|лодка|туризъм
```
Comment (biodranik) [inline data/categories.txt:3611]:
```suggestion
bg:Кораб|лодка
```

File: data/categories.txt [hunk 5]
```diff
@@ -3361,69 +3569,188 @@ zh-Hant:刑柱
...
+vi:Diểm tham quan
+zh-Hans:历史船
+zh-Hant:歷史船
+
 
 historic-tomb|@tourism
 en:Tomb|grave|memorial
-ru:4Гробница|3склеп|захоронение|могила|кладбище
+ar:قبر
+be:Магіла
```
Comment (biodranik) [inline data/categories.txt:3647]:
```suggestion
be:Склеп|магіла
```

**Review by biodranik — CHANGES_REQUESTED** [review]

File: data/categories.txt (hunk 1 above)
Comment (biodranik) [reply data/categories.txt:3125]: Reusing with `ref =` is a good idea.

**Review by jimcarst — APPROVED** [review]

File: data/categories.txt [hunk 6]
```diff
@@ -3046,20 +3121,77 @@ vi:Cổng thành
...
+hu:Városi fal
+id:Tembok Kota
+it:Muro della città
+ja:城壁
+ko:도시의 벽
+mr:शहराची भिंत
+nb:Bymurer
+nl:Stadsmuur
+pl:Mur miejski
+pt:Muralha de cidade
```
Comment (matheusgomesms) [inline data/categories.txt:3148]: Just 'Muralha' is better

File: data/categories.txt [hunk 7]
```diff
@@ -3046,20 +3121,77 @@ vi:Cổng thành
...
+hu:Fort
+id:Benteng
 it:Fortezza
 ja:歴史的な要塞
+ko:요새
 mr:दुर्ग
+nb:Fort
+nl:Fort
 pl:Historyczny fort
 pt:Forte histórico
```
Comment (matheusgomesms) [inline data/categories.txt:3185]: Just 'Forte'

File: data/categories.txt [hunk 8]
```diff
@@ -3212,6 +3345,80 @@ vi:Đài tưởng niệm
...
+hu:Emlékmű
+id:Tugu peringatan
+it:Scultura
+ja:彫刻
+ko:기념비
+mr:शिल्प
+nb:Minnesmerke
+nl:Sculptuur
+pl:Rzeźba
+pt:Escultura memorial
```
Comment (matheusgomesms) [inline data/categories.txt:3372]: Only 'Escultura'

File: data/categories.txt [hunk 9]
```diff
@@ -3212,6 +3345,80 @@ vi:Đài tưởng niệm
...
+hu:Emlékmű
+id:Tugu peringatan
+it:Statua memoriale
+ja:塑像
+ko:기념비
+mr:पुतळा
+nb:Minnesmerke
+nl:Standbeeld
+pl:Pomnik
+pt:Estátua memorial
```
Comment (matheusgomesms) [inline data/categories.txt:3409]: Only 'Estátua'

File: data/strings/types_strings.txt [hunk 10]
```diff
@@ -10361,19 +10345,39 @@
...
+    hu = Történelmi tárgy
+    id = Objek bersejarah
     it = Oggetto storico
     ja = 史跡
+    ko = 역사적 물건
     mr = ऐतिहासिक
+    nb = Historisk gjenstand
     nl = Historisch object
     pl = Historyczne
     pt = Histórico
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:10371]: Objeto histórico

File: data/strings/types_strings.txt [hunk 11]
```diff
@@ -10894,36 +10939,74 @@
...
+    id = Tembok Kota
+    it = Muro della città
+    ja = 城壁
+    ko = 도시의 벽
     mr = शहराची भिंत
+    nb = Bymurer
     nl = Stadsmuur
-    pl = Historyczne mury miasta
+    pl = Mur miejski
     pt = Muralha de cidade
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:10963]: Muralha

File: data/strings/types_strings.txt [hunk 12]
```diff
@@ -10894,36 +10939,74 @@
...
+    hu = Fort
+    id = Benteng
     it = Fortezza
     ja = 歴史的な要塞
+    ko = 요새
     mr = दुर्ग
+    nb = Fort
     nl = Fort
     pl = Historyczny fort
     pt = Forte histórico
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:11000]: Forte

File: data/strings/types_strings.txt [hunk 13]
```diff
@@ -11075,28 +11162,33 @@
...
     id = Tugu peringatan
-    it = Memoriale
+    it = Scultura
     ja = 彫刻
     ko = 기념비
     mr = शिल्प
     nb = Minnesmerke
     nl = Sculptuur
     pl = Rzeźba
     pt = Escultura memorial
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:11186]: Escultura

File: data/strings/types_strings.txt [hunk 14]
```diff
@@ -11124,11 +11221,11 @@
     nl = Standbeeld
     pl = Pomnik
     pt = Estátua memorial
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:11223]: Estátua

File: data/strings/types_strings.txt [hunk 15]
```diff
@@ -11332,11 +11442,11 @@
     nl = Historisch schip
     pl = Okręt
     pt = Barco histórico
```
Comment (matheusgomesms) [inline data/strings/types_strings.txt:11444]: Barco

**Review by matheusgomesms — CHANGES_REQUESTED** [review]
Looks good, just made some fix suggestions in Portuguese.

**Comment by matheusgomesms** [comment]
> thanks @matheusgomesms, I corrected them.

Lovely, thanks!

**Review by matheusgomesms — APPROVED** [review]

**Comment by vng** [comment]
To complete this PR, we should add drawing styles (mapcss) for historic=battlefield.
For example make it the same as historic=memorial.
Otherwise, this type is not included in generated data and search tests failed.

**Comment by vng** [comment]
Ok, now please, make:
- commit with generated strings (./tools/unix/generate_localizations.sh)
- commit with generated styles (./tools/unix/generate_drules.sh)

File: android/res/values-ar/strings.xml [hunk 16]
```diff
@@ -835,6 +835,7 @@
 	<string name="type.barrier.block">حاجز</string>
 	<string name="type.barrier.bollard">عمود حاجز قصير</string>
 	<string name="type.barrier.border_control">أمن الحدود</string>
+	<string name="type.barrier.city_wall">جدار المدينة</string>
```
Comment (biodranik) [inline android/res/values-ar/strings.xml:838]: Was it merged previously, but not regenerated?

**Review by biodranik — APPROVED** [review]

**Pull request merged**